search/v2/geocoder.cpp

```diff
@@ -320,6 +314,8 @@ void Geocoder::DoGeocodingWithLocalities()
   }
 }
 
+double constexpr kComparePoints = MercatorBounds::GetCellID2PointAbsEpsilon();
```

Comment:
s/kComparePoints/kPointCoordsEps/?
Please, move this to the anonymous namespace in the top of the file.